vulnfeeds/cmd/cve-bulk-converter/main.go

```diff
@@ -0,0 +1,114 @@
+// Package main converts Linux CVEs to OSV format in bulk.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/google/osv/vulnfeeds/cmd/cvelist2osv"
+	"github.com/google/osv/vulnfeeds/cves"
+	"github.com/google/osv/vulnfeeds/utility/logger"
+)
+
+type PartialCVE struct {
+	Metadata struct {
+		ID       string `json:"cveId"`
+		Assigner string `json:"assignerShortName"`
+		State    string `json:"state"`
+	} `json:"cveMetadata"`
+}
+
+func main() {
+	// --- Configuration ---
+	years := flag.String("years", "2022,2023,2024,2025", "A comma-separated list of years to process.")
+	workers := flag.Int("workers", 10, "The number of concurrent workers to use for processing CVEs.")
+
+	flag.Parse()
+	var logCleanup = logger.InitGlobalLogger("cvelist-osv", false)
+	defer logCleanup()
+	// --- Main execution ---
+	logger.Infof("Commencing Linux CVE to OSV conversion run")
+
+	repoDir := "cvelistV5"
+
+	// Prepare for processing.
+	localOutputDir := "cvelist2osv"
```

Comment:
These should probably be flags, and the default value can be constants at the top of the file.